smarts/core/road_map.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2020. Huawei Technologies Co., Ltd. All rights reserved.
+# Copyright (C) 2022. Huawei Technologies Co., Ltd. All rights reserved.
```

Comment:
This should be 2021.  (In general, it should be the earliest date the code was -- in something akin to its "present/final form" -- included in the repo.